membership_initial_fee/models/product_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingenieria S.L. - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, fields
+import openerp.addons.decimal_precision as dp
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    initial_fee = fields.Selection(
+        selection=[('none', 'No initial fee'),
+                   ('fixed', 'Fixed amount'),
+                   ('percentage', 'Percentage of the price')],
+        default='none', string="Initial fee", required=True)
+    fixed_fee = fields.Float(digits_compute=dp.get_precision('Product Price'))
```

Comment:
IMHO ```fixed_fee``` should be:

```
fixed_fee = fields.Float(digits_compute=dp.get_precision('Product Price'), related='product_fee.list_price', readonly=True)
```
Because now there are two price fields to configure, one in fee product (fixed_fee) and another in initial fee product (list_price)  and this could be consfusing for user.
 
And shoud have a new boolean field to indicate that product is an initial fee
```
is_initial_fee = fields.Boolean()
```
And configure this domains:

* Show ```membership = True``` and ```is_initial_fee = False``` in Membership Products list
* Show ```membership = True``` and ```is_initial_fee = True``` in 'Product for initial fee' one2many field

By default, when creating a new initial fee product from 'Product for initial fee' one2many field, should have ```membership = True``` and ```is_initial_fee = True``` by default

Then we can see all membership related products (fees and initial fees) in 'Members Analysis' report
 
In case, 'initial_fee' = percentage, IMO current implementation is OK.